app/component/ItineraryListContainer.js

```diff
@@ -203,6 +199,13 @@ class ItineraryListContainer extends React.Component {
     );
   }
 
+  renderMoreButton(arriveBy, position) {
+    if (position === 'top') {
```

Comment:
Same as above, would be nice to have some enum object that contains possible values and compare against that.

Reply:
This jus a local function call parameter. Changed the parameter to boolean.